### DISCUSSION TITLE
How do you do text replacement for emojis

### DISCUSSION THREAD

**Discussion by tallguyjenks** [BryanJenksCommunity/FAQ#182] [Q&A]
i use alfred on MacOS and its snippet feature to make text replacements like:

`,rsq` == 🟥️

there are many applications that perform this functionality on various platforms

a non-exhaustive list being:

- Espanso
- autohotkey
- alfred

**Comment by tantran41** [comment]
bryan, maybe you can do a video on alfred and showed us how to implement this. Thanks!
